### DISCUSSION TITLE
自動平衡摩托車

### DISCUSSION THREAD

**Discussion by yuchien-lin** [1101-GS4719-C/Python-Community-GS4719-C#1875] [General]
對於有些行動不便的人，可以幫助他們更容易的移動，我曾看過有些行動不便的人，他們會在摩托車的後輪多加兩個輔助的輪子，但是這樣行走在馬路上，不方便停車，所以這個自動平衡摩托車能幫助他們更容易的在路上行駛，此外這個摩托車還有倒退的功能，或許對於我們這種用腳就很方便的人來說沒用，但是對於行動不便的人可以說是非常好用的功能。
資料來源:https://technews.tw/2023/12/27/automatic-balancing/

**Comment by vchien03** [comment]
我覺得這個發明非常棒！體積減小也不用擔心平衡的問題，可以倒退真的是很便利的功能，讓整個車體的機動性更高了，甚至還可在無人駕駛使用遠端遙控，需要的時候遙控一下就可以讓摩托車來到你身邊(?)

**Comment by TobywithDino** [comment]
自動平衡聽起來超級高科技ㄟXD 感覺就算真的實現了，應該也很少有人可以負擔的起吧... 但還是希望可以幫助更多行動不變者

**Comment by chuanmin0317** [comment]
感覺對一些人來說，如果機車能夠自動平衡的話就失去那種騎機車的樂趣了，不過對我們來說安全性肯定是上升了，但是我覺得行動不便者還是搭車比較好，畢竟機車還是肉包鐵，不向汽車是鐵包肉

**Comment by lyykey** [comment]
上大學後，我遇到很多不會騎摩托車的同學，甚至連騎腳踏車都很有困難，多數原因都是因為他缺乏平衡感，我相信有了這項設計，多數人都可以擺脫缺乏平衡感的困擾，謝謝同學提供這個資訊~

**Comment by benbebobo149** [comment]
看到這篇文章，我思考的地方是他真正可以利用價值在哪裡。儘管自動平衡摩托車概念聽起來令人振奮，但我不免擔心其潛在的安全隱患。若演算法計算出錯，可能導致使用者受傷，甚至對其他用路人造成生命威脅。除非這項技術具有無可取代的功能，否則我對其發展持保留態度。我認為在技術發展的同時，應該嚴格考慮安全性，並確保能夠有效地解決潛在的風險問題。最後，我要感謝同學的分享，這讓我有機會更深入地思考到這個議題。

**Comment by William08290829** [comment]
我覺得這個發明相當有趣，的確會在路上看到有些人會在旁邊假裝兩個輔助輪，但相對的也佔據了不少空間，但這樣的設計不僅少了原本左右空間，也保有原本平衡的功能，是相當不錯的設計。

**Comment by wanka2030** [comment]
自動平衡摩托車感覺真的很帥欸！如果能配合上自動駕駛，是不是以後大家又都不用去考駕照了（？但騎車還是想要自己有催油門的感覺、或是怒踩踏板加速的滋味，才讓人有著隨風奔馳的畫面😌

**Comment by jim1743** [comment]
我覺得這項發明很有趣，能夠自動平衡的摩托車讓駕駛機車的人的安全性提升，若未來這項技術普及，馬路上的一些事故肯定能夠避免掉，不過我認為，本身行動不便的人還是不要自行駕駛比較好，多搭乘大眾運輸對他們來說比較安全。
